hnix-store-core/src/System/Nix/Path.hs

```diff
@@ -0,0 +1,76 @@
+{-|
```

Comment:
Is this just copied verbatim or any changes along the way?

Reply:
Definitions copied verbatim, but I think I changed the exports list.